### PR TITLE
fix: bump libsession and remove manual unwrapping/wrapping of push()

### PR DESCRIPTION
**Pull request by Bilb**

### CODE REVIEW

File: ts/session/apis/snode_api/swarmPolling.ts [hunk 1]
```diff
@@ -265,9 +266,16 @@ export class SwarmPolling {
...
-      const userConfigMessages = resultsFromAllNamespaces
-        .filter(m => SnodeNamespace.isUserConfigNamespace(m.namespace))
-        .map(r => r.messages.messages);
+      const userConfigMessages = resultsFromAllNamespaces.filter(m =>
+        SnodeNamespace.isUserConfigNamespace(m.namespace)
+      );
+
+      const userConfigMessagesWithNamespace: Array<Array<RetrieveMessageItemWithNamespace>> =
+        userConfigMessages.map(r => {
+          return (r.messages.messages || []).map(m => {
```
Comment (yougotwill) [inline ts/session/apis/snode_api/swarmPolling.ts:275]: lol

File: ts/session/apis/snode_api/swarmPolling.ts [hunk 2]
```diff
@@ -365,58 +373,22 @@ export class SwarmPolling {
...
-          `failed to handle messages hashes "${allMessageHases}" with libsession. Error: "${e.message}"`
-        );
-      }
+    try {
+      window.log.info(
+        `handleConfigMessagesViaLibSession of "${userConfigMessagesMerged.length}" messages with libsession`
+      );
+      await ConfigMessageHandler.handleConfigMessagesViaLibSession(userConfigMessagesMerged);
+    } catch (e) {
+      const allMessageHases = userConfigMessagesMerged.map(m => m.hash).join(',');
```
Comment (yougotwill) [inline ts/session/apis/snode_api/swarmPolling.ts:388]: allMessagesHashes

**Review by yougotwill — CHANGES_REQUESTED** [review]

File: ts/session/apis/snode_api/swarmPolling.ts (hunk 1 above)
Comment (Bilb) [reply ts/session/apis/snode_api/swarmPolling.ts:275]: yeah that sucks, see my discussion in the chat

File: ts/session/apis/snode_api/swarmPolling.ts (hunk 2 above)
Comment (Bilb) [reply ts/session/apis/snode_api/swarmPolling.ts]: fixed

**Review by yougotwill — APPROVED** [review]
LGTM 🚢

**Pull request merged**